src/protocols/multi_party_ecdsa/gg_2020/state_machine/sign/rounds.rs

```diff
@@ -147,20 +147,34 @@ impl Round1 {
         let i = usize::from(self.i - 1);
         for j in 0..ttag - 1 {
             let ind = if j < i { j } else { j + 1 };
+
             let (m_b_gamma, beta_gamma, _beta_randomness, _beta_tag) = MessageB::b(
                 &self.sign_keys.gamma_i,
                 &self.local_key.paillier_key_vec[l_s[ind]],
                 m_a_vec[ind].clone(),
                 &self.local_key.h1_h2_n_tilde_vec,
             )
-            .expect("Incorrect Alice's range proof in MtA");
+            .map_err(|e| {
+                Error::Round1(ErrorType {
+                    error_type: e.to_string(),
+                    bad_actors: vec![],
```

Comment:
We need to add the bad actor here. One of these proofs fails and we aren't returning that party's index.